nni/algorithms/compression/v2/pytorch/base/compressor.py

```diff
@@ -0,0 +1,282 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Why do have a father class named `base`? what is `base` for?

Reply:
The `base` is for some base classes like `Compressor` `Pruner` `Quantizer` and some base classes used by `Pruner` like `DataCollector`...